tests/common/type_list.h

```diff
@@ -164,31 +164,51 @@ struct arrow_operator_overloaded {
 
 // Returns instance of type T
 template <typename T>
-inline constexpr auto get_init_value_helper(int x) {
-  return x;
-}
+struct init_value_helper {
+  static constexpr auto get(int x) { return static_cast<T>(x); }
+};
+
+// Returns instance of type T
```

Comment:
"instance of type T" or "instance of type vec<T, N>"?
The same applies to the comment below.